app/controllers/RecommendationController.php

```diff
@@ -18,6 +18,8 @@ function __construct(SettingRepository $settings, RecommendationForm $recommenda
     $this->beforeFilter('isClosed');
     // Check that the current user doesn't create many applications
     $this->beforeFilter('createdRec', ['only' => ['create']]);
+    // check that the user is logged in
+    $this->beforeFilter('auth');
```

Comment:
i think this will actually block recommenders from editing and submitting since they don't have to be logged in, only the applicant.